tests/python/requirements/PySafetyBearWithoutMockTest.py

```diff
@@ -0,0 +1,25 @@
+from queue import Queue
+from bears.python.requirements.PySafetyBear import PySafetyBear
+from coalib.settings.Section import Section
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+
+
+class PySafetyBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.uut = PySafetyBear(Section('name'), Queue())
+
+    def test_without_vulnerability(self):
+        self.check_validity(self.uut, ['lxml==3.6.0'])
+
+    def test_with_vulnerability(self):
+        self.check_invalidity(self.uut, ['bottle<0.12.10', 'bottle==0.10.1'])
```

Comment:
Why two here?
(c.f. https://github.com/coala/coala-bears/pull/2113#discussion_r150035723 )

Reply:
```
>>> bottle = package('bottle', '0.10.1')
>>> x = [bottle]
>>> check(x)
[Vulnerability(name='bottle', spec='<0.12.10', version='0.10.1', 
data={'id': 'pyup.io-25642', 'specs': ['<0.12.10'], 'v': '<0.12.10', 'cve': 'CVE-2016-9964',
 'advisory': 'redirect() in bottle.py in bottle 0.12.10 doesn\'t filter a "\\r\\n" sequence, which leads to a CRLF attack, as demonstrated by a redirect("233\\r\\nSet-Cookie: name=salt") call.'
} ) ]
```
One is `spec` and second is its `version` , Its not two different package,

Reply:
That explains what you are doing, but it is wrong.  Please understand what `check_invalidity` is doing.

Reply:
I will make some tests in which I consider good and bad files to check things , I will change it totally